README.md

```diff
@@ -37,27 +50,37 @@ We test this foreign data wrapper on Ubuntu Server 18.04 using PostgreSQL-11 tog
   sudo ldconfig
   ```
 
-- Build this foreign data wrapper with RocksDB:
+- For users who want to build this foreign data wrapper with [RocksDB](https://github.com/facebook/rocksdb):
   
   ```sh
+  git clone https://github.com/vidardb/PostgresForeignDataWrapper.git
+
   cd PostgresForeignDataWrapper 
 
   make
 
   sudo make install
   ```
 
-- Build this foreign data wrapper with VidarDB:
+- For users who want to build this foreign data wrapper with [VidarDB](https://github.com/vidardb/vidardb):
 
   ```sh
+  git clone https://github.com/vidardb/PostgresForeignDataWrapper.git
```

Comment:
Do you think reducing the duplication between building with VidarDB and RocksDB is a good idea? I mean, there is only one line different: `make VIDARDB=true`. Why we have these same code here? Can we come up with a better way to describe this?